packages/core/src/Synchronizer.ts

```diff
@@ -45,13 +45,17 @@ export class Synchronizer extends EventEmitter {
 
     private pollId: string | null = null
     public pollRate: number = 5000
-    public blockRate: number = 100000
+    public blockRate: number = 20
```

Comment:
this value can be set higher e.g. `100000`
or the memory will be run out easily
```shell
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```

Reply:
fixed